engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -195,13 +199,21 @@ private void initRenderingSupport() {
         shaderManager.initShaders();
         postProcessor.initializeMaterials();
         initMaterials();
+
+        shadowMap = new ShadowMapNode(getMaterial("engine:prog.shadowMap"),
+                playerCamera,
+                this,
+                backdropProvider,
+                renderingConfig,
+                graphicState);
+
+        dag.add(shadowMap);
```

Comment:
Let's call the instance a shadowMapNode. This RenderNode is not the shadow map itself.